<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Mhiremath-L" name="Test" tests="2" failures="1" timestamp="14 Feb 2020 17:09:13 GMT" time="7.508" errors="0">
  <testcase name="testapi" time="3.902" classname="com.api.testng.UsersApi.GetAPI"/>
  <testcase name="numberpost" time="3.592" classname="com.api.testng.UsersApi.PostAPI">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;DB returned 23505&amp;gt; but was:&amp;lt;Succcess&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<DB returned 23505> but was:<Succcess>
	at com.api.testng.UsersApi.PostAPI.numberpost(PostAPI.java:42)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- numberpost -->
</testsuite> <!-- Test -->
